modules/constructors/snowflake/test/integration/ST_BEZIERSPLINE.test.js

```diff
@@ -8,7 +8,7 @@ test('ST_BEZIERSPLINE should work', async () => {
     const rows = await runQuery(query);
     expect(rows.length).toEqual(1);
     expect(JSON.stringify(rows[0].BEZIERSPLINE1)).toEqual('{"coordinates":[[121.025390625,-22.91792293614603],[122.40878796386718,-22.04579264902266],[125.672291015625,-20.380801787862705],[129.4858663330078,-19.350050352981388],[132.704296875,-20.01569558887108],[135.25804138183594,-21.75113289066844],[137.26212890625,-24.05282839437402],[138.7404580078125,-26.51113113177313],[139.065328125,-29.074453813709248],[138.68492871093753,-31.163421738140798]],"type":"LineString"}');
-    expect(JSON.stringify(rows[0].BEZIERSPLINE2)).toEqual('{"coordinates":[[-6,-0.5],[-3.6649305555555554,0.4259259259259258],[-1.0611111111111116,-0.24074074074074053],[1.5,0],[4.06111111111111,0.24074074074074117],[6.664930555555557,-0.42592592592592565]],"type":"LineString"}');
+    expect(JSON.stringify(rows[0].BEZIERSPLINE2)).toEqual('{"coordinates":[[-6,-0.5],[-3.6649305555555554,0.4259259259259258],[-1.0611111111111116,-0.24074074074074053],[1.5,0],[4.06111111111111,0.24074074074074117],[6.664930555555557,-0.42592592592592565],[9,0.5]],"type":"LineString"}');
```

Comment:
I guess this is unrelated, but do you know what's going on there?

Reply:
Yes this is a totally unrelated topic. This test was crashing before the changes and we didn't notice. ST_BEZIERSPLINE was returning a line that wasn't reaching the end of the input line.
<img width="741" alt="Captura de pantalla 2021-11-10 a las 9 36 10" src="https://user-images.githubusercontent.com/6054336/141078499-fac74a36-5bc2-4b0b-be16-9897af4d51c4.png">
But now it does.
<img width="730" alt="Captura de pantalla 2021-11-10 a las 9 36 57" src="https://user-images.githubusercontent.com/6054336/141078681-5b16b375-f832-4332-bb99-cf9250ac4153.png">